arch/xtensa/src/esp32s3/esp32s3_ble_adapter.c

```diff
@@ -717,7 +754,7 @@ static void interrupt_handler_set_wrapper(int intr_num, void *fn, void *arg)
   adapter->arg = arg;
 
   ret = irq_attach(irq, esp_int_adpt_cb, adapter);
-  DEBUGASSERT(ret == OK);
+  DEBUGVERIFY(ret);
```

Comment:
In general change is not needed because the actual call is done just above. We can change it to put function call into `DEBUGVERIFY` and remove `ret` varibale

Reply:
It makes sense!